Frontend/src/cse-ui-kit/events_container/EventsContainer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
being a little pedantic about the naming of this file, not sure if EventsContainer is a good name for this component if we want it to be a reusable component. Maybe something like `TransparentGlassContainer` etc. think of a good name, something that better describes what it looks like instead of EventsContainere